tyk-mdcb-data-plane/values.yaml

```diff
@@ -249,8 +252,8 @@ tyk-gateway:
       tags: ""
 
     # used to decide whether to send the results back directly to Tyk without a hybrid pump
-    # if you want to send analytics to control plane instead of pump, change value to "rpc"
-    analyticsConfigType: ""
+    # if you want to use a hybrid pump, please change "rpc" to ""
+    analyticsConfigType: "rpc"
```

Comment:
i guess we do not need to change this. what do you think? can you please double-check?

Reply:
it was due to a merge conflict, fixed